phpBB/adm/style/acp_groups.html

```diff
@@ -113,7 +113,7 @@ <h3>{L_WARNING}</h3>
 		<legend>{L_GROUP_AVATAR}</legend>
 	<dl>
 		<dt><label>{L_CURRENT_IMAGE}{L_COLON}</label><br /><span>{L_AVATAR_EXPLAIN}</span></dt>
-		<dd>{AVATAR}</dd>
+		<dd>{% if AVATAR_HTML %}{{ AVATAR_HTML }}{% else %}<img src="{{ ADMIN_ROOT_PATH ~ 'images/no_avatar.gif' }}" alt="">{% endif %}</dd>
```

Comment:
If I'm not wrong, this constant don't work with routes, you have to use web_root_path

Reply:
It's set in the [adm_page_header](https://github.com/phpbb/phpbb/blob/master/phpBB/includes/functions_acp.php#L80), and seeing currently there are no routes and controllers possible within in control panels, this should be fine. And once CP's are moved, these template variables will have to be updated anyway, so then it will still work.